src/components/core/TruncatedText/TruncatedText.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Base = ({ is: Component, ...props }) => <Component {...props} />;
+
+/**
+ * Truncate text with an ellipsis.
+ */
+const TruncatedText = styled(Base)`
```

Comment:
I think it needs `display: inline-block`

Reply:
It works without, though I think you could be right ;-/

Reply:
It works because of div I think. Try with a span.